em++.py

```diff
@@ -6,12 +6,19 @@
 
 import sys
 import emcc
+import glob
 
 emcc.run_via_emxx = True
 
 if __name__ == '__main__':
   try:
-    sys.exit(emcc.main(sys.argv))
+    argv = []
+    for i in sys.argv:
+      if '*' in i:
+        argv.extend(glob.glob(i))
+      else:
+        argv.append(i)
+    sys.exit(emcc.main(argv))
```

Comment:
Isn't it up to the shell to do this for you?

CC @RReverser ?

Reply:
It's shell-dependant, Linux shells (bash, fish) tend to expand wildcards early, but cmd and PowerShell don't.

This way Windows shells don't need xargs to deal with very long lists of arguments as wildcard is passed to the program as-is, but it does mean the responsibility for expansion is on programs themselves.